webhook.py

```diff
@@ -31,8 +27,8 @@ class WebhookMessage():
     """Represents a Discord webhook message
     """
     embeds: List[WebhookEmbed]
-    username: str = WEBHOOK_USERNAME
-    avatar_url: str = WEBHOOK_AVATAR_URL
+    username: str = os.environ["WEBHOOK_USERNAME"]
+    avatar_url: str = os.environ["WEBHOOK_AVATAR_URL"]
```

Comment:
Should be single quotes
```suggestion
    username: str = os.environ['WEBHOOK_USERNAME']
    avatar_url: str = os.environ['WEBHOOK_AVATAR_URL']
```